trade-finance-manager-api/src/graphql/reducers/mappings/facilities/mapTenor.js

```diff
@@ -3,23 +3,29 @@ const { findLatestCompletedAmendment } = require('../../../helpers/amendment.hel
 
 // maps tenor from new amendment coverEndDate or from original facility
 const mapTenor = (facilitySnapshot, facilityTfm, facility) => {
-  const { facilityStage, ukefGuaranteeInMonths: monthsOfCover } = facilitySnapshot;
+  // Facility stage
+  const { hasBeenIssued } = facilitySnapshot;
+  /**
+   * `monthsOfCover`: GEF
+   * `ukefGuaranteeInMonths`: BSS/EWCS
+   */
+  const months = facilitySnapshot.ukefGuaranteeInMonths ?? facilitySnapshot.monthsOfCover;
+  // If issued
   const { exposurePeriodInMonths } = facilityTfm;
+  let exposurePeriod = exposurePeriodInMonths;
 
-  // sets original exposure period from facility
-  let updatedExposurePeriodInMonths = exposurePeriodInMonths;
   // if amendments
   if (facility?.amendments?.length) {
     const latestAmendmentTFM = findLatestCompletedAmendment(facility.amendments);
 
     // checks if exposure period in months in latest completed amendment
     if (latestAmendmentTFM?.amendmentExposurePeriodInMonths) {
       // sets updatedExposurePeriodInMonths as amendment exposure period
-      updatedExposurePeriodInMonths = latestAmendmentTFM.amendmentExposurePeriodInMonths;
+      exposurePeriod = latestAmendmentTFM.amendmentExposurePeriodInMonths;
```

Comment:
The comment above says `updatedExposurePeriodInMonths` instead of `exposurePeriod`

is this change expected? I'm not familiar - is `updatedExposurePeriodInMonths` a field that exists?

Reply:
I have only changed the variable name to match DB field. `updatedExposurePeriodInMonths` does not exists in DB.